Source/Views/StackViewController.swift

```diff
@@ -48,13 +54,13 @@ final class StackViewController: UIViewController {
         scrollView.alwaysBounceVertical = false
         scrollView.showsHorizontalScrollIndicator = false
         
-        scrollView.snp.makeConstraints { (make) in
+        scrollView.snp.makeConstraints { make in
```

Comment:
@MP0w sorry, I have a code format plugin installed. (https://github.com/nicklockwood/SwiftFormat)
I tried to only use it on my code, but I case I used it on the whole file.
In our codebase we would remove the brackets, if they are not needed. I can revert these changes, if you want to.